cfme/fixtures/ansible_fixtures.py

```diff
@@ -164,50 +165,47 @@ def ansible_service_catalog(appliance, ansible_catalog_item, ansible_catalog):
     return service_catalog
 
 
-@pytest.fixture(scope="function")
-def ansible_service_request_funcscope(appliance, ansible_catalog_item):
-    request_descr = "Provisioning Service [{0}] from [{0}]".format(ansible_catalog_item.name)
-    service_request = appliance.collections.requests.instantiate(description=request_descr)
-    yield service_request
-
-    if service_request.exists():
-        service_id = appliance.rest_api.collections.service_requests.get(description=request_descr)
-        appliance.rest_api.collections.service_requests.action.delete(id=service_id.id)
+def bulk_service_teardown(appliance):
+    """Delete all service requests on the appliance via rest"""
+    # big diaper here because of service requests having the same description
+    requests = [r for r in appliance.rest_api.collections.service_requests]
+    if requests:
+        delete_resources_from_collection(
+            resources=[r for r in appliance.rest_api.collections.service_requests],
+            collection=appliance.rest_api.collections.service_requests,
```

Comment:
Any particular reason for re-generating the list of requests here instead of passing `resources=requests`?

Reply:
ah! missed cleaning this up in the re-suffle. Have to check it first otherwise we hit an exception - but you're right, no need to generate the list twice.